projects/packages/jetpack-mu-wpcom/src/features/help-center/class-help-center.php

```diff
@@ -105,16 +105,20 @@ function ( $wp_admin_bar ) {
 		}
 
 		if ( $variant !== 'wp-admin-disconnected' && $variant !== 'gutenberg-disconnected' ) {
-			// Load translations directly from widgets.wp.com.
-			wp_enqueue_script(
-				'help-center-translations',
-				'https://widgets.wp.com/help-center/languages/' . Common\determine_iso_639_locale() . '-v1.js',
-				array( 'wp-i18n' ),
-				$version,
-				true
-			);
-
-			$script_dependencies[] = 'help-center-translations';
+			$locale = Common\determine_iso_639_locale();
+
+			if ( 'en' !== $locale ) {
+				// Load translations directly from widgets.wp.com.
+				wp_enqueue_script(
+					'help-center-translations',
+					'https://widgets.wp.com/help-center/languages/' . Common\determine_iso_639_locale() . '-v1.js',
```

Comment:
Maybe reuse `$locale` var here?

Reply:
Oops missed that one. I almost made this a class var.